### PR TITLE
Added support to install different pipeline version based on openshift version

### PR DESCRIPTION
**Pull request by tarukumar**
…ersion

### CODE REVIEW

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot [hunk 1]
```diff
@@ -57,6 +58,10 @@
 
 Install Red Hat OpenShfit Pipelines
     [Documentation]    Install Red Hat OpenShfit Pipelines
+    ${oc_vesrion}    Get OpenShift Version
+    ${pipeline}   Get From Dictionary     ${OPENSHIFT_PIPELINES_VERSION}    ${oc_vesrion}
+    ${return_code}    ${output}    Run And Return Rc And Output    sed -i "s,channel: .*,channel: ${pipeline},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot:63]: Add the following `    # robocop: disable:line-too-long`

**Review by diegolovison — DISMISSED** [review]

**Comment by tarukumar** [comment]
4.12- https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1726/ 
4.11-  https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1727/

**Review by bdattoma — APPROVED** [review]

**Review by mwaykole — APPROVED** [review]

**Comment by tarukumar** [comment]
4.10 :https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1730/ installation worked fine but it looks like problem from the cluster that TC is failed due to cluster issue.

**Comment by diegolovison** [comment]
The test is failing.

**Pull request merged**